### ISSUE TITLE
Move to the workspace where a detected application is opened

### ISSUE THREAD

**Issue by Anastand** [nikitabobko/AeroSpace#447]
**i want to be able to move to the new workspace when i open app with rules**
- DEFAULT
[[on-window-detected]]
if.app-id = 'com.github.wez.wezterm'
run = "move-node-to-workspace T"

- i want to be able to do the following
[[on-window-detected]]
if.app-id = 'com.github.wez.wezterm'
run = ["move-node-to-workspace T" ,"wokspace T"]

## Checklist

- [x] I've searched for duplicates. My search queries were (please don't forget to search in closed issues too):
  - move if detected
  - if detected 
  - ...
- [x] I've read the documentation https://nikitabobko.github.io/AeroSpace/guide & https://nikitabobko.github.io/AeroSpace/commands
```
aerospace version
aerospace CLI client version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
```

**Comment by crates51** [comment]
Wierd, for me this

```
[[on-window-detected]]
if.app-id = 'company.thebrowser.Browser'
run = ['move-node-to-workspace 2']
```
is moving my browser to workspace 2 AND moves me to workspace 2 automatically

**Comment by Anastand** [comment]
@crates51  yeah i dont know , there are not many people who are using it so maybe there are some bugs

**Comment by nikitabobko** [comment]
The issue doesn't make sense

As correctly pointed by @crates51, inside the callback, move-node-to-workspace works as if "focus followed window". But normally, move-node-to-workspace doesn't have "focus following" behavior 

The reason on why it works this way is actually not because move-node-to-workspace has "focus following" inside the callback, but because newly appeared window steals the focus from AeroSpace.

It's an implementation coincidence that AeroSpace firstly runs the callback and then checks if the focus was stolen by macOS.

But it's an implementation coincidence that I'm not going to change, because it's a good naive expected behavior, which too many people already rely on
